yaksh/models.py

```diff
@@ -68,21 +63,29 @@
                 ('completed', 'Completed'),
               )
 
+
 def get_assignment_dir(instance, filename):
-    return '%s/%s/%s' % (instance.user.user, instance.assignmentQuestion.id, filename)
+    return os.sep.join((
+        instance.user.user, instance.assignmentQuestion.id, filename
```

Comment:
This was already there and I did not change this, I just changed the implementation a bit.  We can change this later if needed.